radicle-node/src/wire.rs

```diff
@@ -129,6 +129,8 @@ impl Encode for PublicKey {
 
 impl<const T: usize> Encode for &[u8; T] {
     fn encode<W: io::Write + ?Sized>(&self, writer: &mut W) -> Result<usize, io::Error> {
+        // TODO: This can be removed when the clippy bugs are fixed
+        #[allow(clippy::explicit_auto_deref)]
```

Comment:
Any idea which one? Linking would make it easy for others to know when to remove the fix

Reply:
I think it's this: https://github.com/rust-lang/rust-clippy/issues/9710

already merged so should find its way soon.